modules/ossm-migrating-a-cluster-wide-deployment-using-the-simple-migration-method.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-a-cluster-wide-deployment-using-the-simple-migration-method_{context}"]
+= Migrating a cluster-wide deployment using the simple migration method 
+
+.Prerequisites
+
+* You have deployed {ocp-product-title} 4.14 or later.
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role.
+* You have completed the premigration checklists.
+* You have the {SMProduct} {SMv2Version} Operator installed.
+* You have the {SMProduct} 3 Operator installed.
+* You have created an `IstioCNI` resource.
+* You have installed the `istioctl` tool.
+* You are running a cluster-wide Service Mesh control plane resource.
+* You have installed the `bookinfo` application.
+
+.Procedure
+
+. Identify the namespaces that contain a 2.6 control plane by running the following command:
++
+[source,terminal]
+----
+$ oc get smcp -A
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE      NAME                   READY   STATUS            PROFILES      VERSION   AGE
+istio-system   install-istio-system   6/6     ComponentsReady   ["default"]   2.6.6     115m
+----
+
+. Create a YAML file named `ossm-3.yaml`. This procedure creates the {istio} resource for the 3.0 installation in the same namespace as the `ServiceMeshControlPlane` resource for the 2.6 installation.
++
+[NOTE]
+====
+In the following example configuration, the {istio} control plane has access to all namespaces on the cluster. If you want to limit the namespaces the control plan has access to, you must define discovery selectors. You must match all the data plane namespaces that you plan to migrate from version 2.6.
+====
++
+.Example `Istio` resource
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+  name: ossm-3 # <1>
```

Comment:
I think this actually _needs_ to be named `default` for the simple method per the instructions here: https://github.com/openshift-service-mesh/sail-operator/tree/main/docs/ossm/ossm2-migration/cluster-wide#simple-migration-of-26-installation-with-istio-injectionenabled-label.

Either the `Istio` needs to be named `default` or there needs to be a `default` `IstioRevisionTag`, right @FilipB?

Reply:
@FilipB I've changed the name in the YAML example.

I'm wondering if I should also change things like the statement "Create a YAML file named `ossm-3.yaml`." to Create a YAML file named `default.yaml`?

And if I do, then also update the syntax to apply the YAML file.

What do you think? Leave those as they are or change them, too?

Reply:
We can leave as is. Done.